endorsement/storeapi.proto

```diff
@@ -43,6 +43,76 @@ message OpenArgs {
 
 message CloseArgs {
 }
+message SwComponentID {
+    common.TokenFormat type = 1;
+    google.protobuf.Struct id = 2;
+}
+// SwComponentInfo defines the structure of a single Software Component 
+// identified by a SoftwareComponent Identifer.
+message SwComponentInfo {
+    SwComponentID comp_id = 1;
+    google.protobuf.Struct comp_meta = 2; 
+}
+
+message AddSwComponentArgs {
+    repeated SwComponentInfo info = 1;
+}
+
+message GetSwComponentArgs {
+    SwComponentID id = 1;
```

Comment:
In a previous comment you said _"There is already an "id" inside the SwComponentID, hence this way of representation!"_ which seems to apply here as well, right?  Should we call this `comp_id` as we do in `SwComponentInfo`?

Reply:
Good Point, I can change this to be consistent!